mail_optional_autofollow/wizard/mail_compose_message.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super(MailComposeMessage, self).default_get(fields_list)
+        if self.env.context.get('mail_post_autofollow'):
+            res['autofollow_recipients'] = True
+        return res
+
+    autofollow_recipients = fields.Boolean()
```

Comment:
I think you need a more user friendly string here instead of 'Autofollow recipients'. How about 'Make recipients followers' or something like that? In any case, I think you should add a help text detailing what the checkbox does.